debian/postinst.d/14-python-packages.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+main() {
+    env python3.9 -m pip install -r /usr/share/ottopia-keepalived-exporter/requirements.txt
```

Comment:
we need to be sure that the installation of the packages is system-wide and not into specific user python directory